templates/single.twig

```diff
@@ -84,14 +84,28 @@
 
 		<!-- Post Block Start -->
 		<div class="container">
-			<div class="post-content">
+			<div class="post-content narrow-container">
```

Comment:
Perhaps it makes more sense to replace the outer `container` `div` with this class?

Reply:
That would not work on mobile

Reply:
I see. Would using both classes work?
```
<div class="container narrow-container">

Reply:
That results in a small padding difference, which messes up the alignment 😕

Reply:
Can we fix that with further changes? Or would it be too many changes?

Reply:
I would rather address this later on, it's already a few unrelated changes with the addition of the `narrow-container` class and they impact quite a lot of pages

Reply:
I don't see how these changes are "unrelated"... We're trying to build something that makes sense, and putting an extra container class, and then nesting it inside of the existing one doesn't really make sense to me.

Reply:
"Unrelated" in the sense that it wasn't necessary for the ticket itself and it could have been done separately... The changes we made so far already simplify the code, but as I said they will impact a lot of pages so I'm a bit hesitant to keep changing things right now

Reply:
[Updated](https://github.com/greenpeace/planet4-master-theme/pull/1630/commits/228ee5e710b7a4d3b6aaad7003b54d838b7d98d4)!